README.md

```diff
@@ -467,6 +467,16 @@ Manages the GPG keys that Apt uses to authenticate packages.
 
 This module is tested and officially supported on Debian 6 and 7 and Ubuntu 10.04, 12.04, and 14.04. Testing on other platforms has been light and cannot be guaranteed.
 
+This module is not designed to be split across stages. If you want to ensure that apt update is run before all packages, add a proper dependency like this:
+
+~~~puppet
+Class['apt::update'] -> Package<||>
```

Comment:
I'm not sure I follow. Using the spaceship collector without specifying the provide will collect all package resources in the catalogue, which you probably don't need to do, as it will collect gems, npm packages etc.